docs/development/bump-k8s-go.md

```
@@ -0,0 +1,466 @@
+## Step 6: Validate
+
+```bash
+make build
+make test
+make fmt
+make vet
+```
+
+Fix any failures before proceeding.
+
+## Step 6: Validate
```

markdownlint-cli2 (0.23.0)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)